src/pp.ml

```diff
@@ -96,9 +94,7 @@ module Render = struct
     | Verbatim x -> pp_print_string ppf x
     | Char x -> pp_print_char ppf x
     | Break (nspaces, shift) -> pp_print_break ppf nspaces shift
-    | Wrapping_break (nspaces, shift) ->
-      pp_print_custom_break ppf ~fits:("", nspaces, "")
-        ~breaks:(" \\", shift, "")
+    | Custom_break (fits, breaks) -> pp_print_custom_break ppf ~fits ~breaks
```

Comment:
Could you replace `Break` by an instance of `Custom_break`? And I guess simply call it simply `Break`. That will make implementing the other version I have in mind simpler.

Looking at format.ml, this is also how `pp_print_break` is implemented, i.e. it is calling `pp_print_custom_break`